templates/taxi/driver_list.html

```diff
@@ -6,8 +6,14 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          {% if driver.id == user.id %}
```

Comment:
Code Efficiency: The if-else block for checking if the driver is the current user could be simplified. You could consider using Django's `ifequal` tag to compare `driver.id` and `user.id` directly. This would make the code more concise and easier to read.